tests/Resources/Page/ODH/AiApps/Aikit.robot

```diff
@@ -26,4 +26,6 @@ Verify JupyterHub Can Spawn AIKIT Notebook
     Wait Until Page Contains Element    xpath://input[@name="oneAPI AI Analytics Toolkit"]
```

Comment:
can you add the new ODS polarion ID to this test case?

Reply:
Yes I have added in the test where this keyword is used